lib/Serge/Application.pm

```diff
@@ -24,12 +25,19 @@ sub new {
 sub run {
     my ($self) = @_;
 
-    my ($help, $debug);
+    my ($help, $debug, $version);
     my $result = GetOptions(
         'help' => \$help,
         'debug' => \$debug,
+        'version|V' => \$version,
     );
 
+    # print out version when passing the flat to a bare `serge` command
+    if ($version && !@ARGV[0]) {
```

Comment:
I am unsure if there is a better (and more reliable) way than `!@ARGV[0]` to determine whether any commands have been passed.